subsys/zigbee/lib/zigbee_logger_ep/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig ZIGBEE_LOGGER_EP
+	bool "Enable Zigbee endpoint handler for logging incoming ZCL frames"
+	imply ZIGBEE_HELPERS
```

Comment:
Will it work w/o Zigbee helpers? If not, I think this module should rather `select` it.